static/json/languages/en_US.json

```diff
@@ -1382,6 +1382,7 @@
     "not_available": "Price not available",
     "more_options": {
       "copy_address": "Copy Address",
+      "view_on_etherscan": "View on Etherscan",
```

Comment:
I'd use the key `view_on_explorer` for consistency and accept the explorer name (i.e. Etherscan vs Arbiscan)